Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN mkdir /srv/ttn-lorawan/public/blob
 
 VOLUME ["/srv/ttn-lorawan/public/blob"]
 
+HEALTHCHECK --interval=1m --timeout=5s CMD curl -f http://localhost:1885/health/live || exit 1
```

Comment:
Probably add a note that this only works if the listen port is unchanged

Reply:
Can't we `ENV` this?

Reply:
Should we dedicate a fixed port to this otherwise?

Reply:
It's not only the port, it's also the availability of the non-TLS listener

Reply:
I also think a dedicated port makes sense.